rosdep/python.yaml

```
@@ -8226,6 +8226,14 @@ python3-smc-pip:
   ubuntu:
     pip:
       packages: [smc]
+python3-socketio:
+  debian:
+    bullseye: [python3-socketio]
+  fedora:
+    packages: [python3-socketio]
```

This isn't the right structure. It would be defining a rule for a Fedora version called `packages`, which is obviously not right.
```suggestion
  fedora: [python3-socketio]
```